src/ChainSafe.GamingSDK.EVM.Tests/ChainsafeRPCTests.cs

```diff
@@ -1,67 +1,93 @@
             */
+
+            // Deploy a contract with https://remix.ethereum.org/ and your Ganache instance.
+            // Make sure to select Ganache Provider as the Environment. Once deployed, replace the _contractAddress value with your contract address
+            // todo: This is impossible to automate
+            /* contractAddress = "0x2061c2B28F007DD0eD5064A61d352521CC1d2827"; */
```

Comment:
Let's not do this please. Use ganache cli to start ganache cli to start local blockchain and use nethereum to deploy contract before tests^^

Reply:
I have never seen any good coming from starting CLI stuff in tests. However, the contract deployment step should indeed be automated.

Reply:
I don't know any other way to start local blockchain in C#. It's kinda common practice with blockchain to start dev node locally before tests and close it afterwards

Reply:
I'm not saying that you have to run cli from c#, having some script that starts ganache, runs tests and then closes ganache is fine also^^

Reply:
I generally prefer including instructions with the tests: "A local Ganache node must be running on localhost:xxxx for these tests to run" It's also simpler to set up CI that way. The script would have to be able to locate the CLI, and you'd have to mess with PATH and stuff.